Adds a separate docs page for istioctl. (#4274)
istio/istio.io: content/docs/setup/kubernetes/getting-started/index.md

    $ export PATH=$PWD/bin:$PATH
    {{< /text >}}

> 1. You can enable the [auto-completion option](/docs/ops/setup/istioctl/) when working with a bash or ZSH console in a Mac or Linux system.

Reviewer: c/in/on?  And I'm not up to date on Windows, but doesn't it have some sort of Linux shell that this might work on as well? Maybe just end with a period after console?

Author: I just ended with the period after console.

> 1. If you are using Bash or ZSH, you have the option of [Enabling auto-completion](/docs/ops/setup/istioctl/) for your macOS or Linux system.

Reviewer 2: a numbered list doesn't make sense here as there is only one item.  You might try a tip callout instead.

Author: > a numbered list doesn't make sense here as there is only one item. You might try a tip callout instead.

actually, when this page is rendered, it ends up being step #4 in the list of steps for "Downloading the release" on the Getting Started page- @frankbu specifically requested that we add a step 4 here to give us a link from the Getting Started page to the separate istioctl auto-completion page.

> 1. If you are using Bash or ZSH, you have the option of [Enabling auto-completion](/docs/ops/setup/istioctl/) for your macOS or Linux system.

Reviewer 3: This reads a little awkward.

@rcaballeromx, what would you suggest?

Author: @frankbu perhaps it does. I'll see if we get any suggestions from @rcaballeromx ...

> 1. If you are using Bash or ZSH, you have the option of [Enabling auto-completion](/docs/ops/setup/istioctl/) for your macOS or Linux system.

Reviewer 4: I suggest you change the text to:
```suggestion
1. You can enable the [auto-completion option](/docs/ops/setup/istioctl/) when working with a bash or ZSH console in a Mac or Linux system.
```
